app-tasks/Dockerfile

```diff
@@ -14,13 +14,18 @@ RUN set -ex \
     && buildDeps=' \
        python-dev \
     ' \
+    && javaDeps=' \
+       openjdk-8-jre \
+    ' \
     && gdal=' \
        gdal-bin \
        libgdal1h \
        libgdal-dev \
     ' \
     && echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list \
-    && apt-get update && apt-get install -y --no-install-recommends ${buildDeps} ${gdal} \
+    && apt-get update \
+    && apt-get -t jessie-backports install -y --no-install-recommends  \
```

Comment:
It doesn't look like this has a package listed, should this specify `ca-certificates-java`?